### PR TITLE
Switch flag so that view operation resolves the service broker URI instead of the service instance

### PR DESCRIPTION
**Pull request by ojhughes**

### CODE REVIEW

File: instance/view.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func (vo *viewOperation) Run(serviceInstanceAdminURL string, accessToken string)
 }
 
 func (vo *viewOperation) IsLifecycleOperation() bool {
-	return false
+	return true
```
Comment (raulavila) [inline instance/view.go]: This makes me think if the name `IsLifeCycleOperation` is accurate. Initially only `stop`, `start`, `restage` and `restart` were considered lifecycle operations but `view` doesn't seem to fit well...Not sure if we can find a better name (`useServiceBrokerUriInV3`?)

Comment (ojhughes) [reply instance/view.go]: `IsServiceBrokerEndpoint` perhaps?

Comment (raulavila) [reply instance/view.go]: Could be, but it only applies in the context of version 3. But at least it's better than the current name after your change, so happy to use that one :)

Comment (ojhughes) [reply instance/view.go]: The statement is true none the less :-)

**Pull request merged**